api/autopilot.go

```diff
@@ -29,6 +29,7 @@ type (
 	// HostsConfig contains all hosts configuration parameters.
 	HostsConfig struct {
 		IgnoreRedundantIPs bool                        `json:"ignoreRedundantIPs"`
+		MaxDowntimeHours   uint64                      `json:"maxDowntimeHours"`
```

Comment:
Why not `ParamDuration` or `time.Duration`?

Reply:
No reason, to be consistent with the other config values like `period` and `renewWindow`. I can make it a `time.Duration`? `ParamDuration` doesn't make sense here imo.

Reply:
`time.Duration` would be great

Reply:
I added `ParamDurationHour` so we can use it in the `bus` API. I think it's really weird though to use it in the config so I would keep it as a simple uint64 I don't see the value in implementing custom marshalers for something we can easily represent as a uint.